exercises/react/example.cr

```diff
@@ -0,0 +1,80 @@
+alias CellValue = Int32
```

Comment:
The tests only act on ints. However, you could imagine that the types of the cells could be generic if we want to add more to this problem. Not sure if we want to be doing that.